cli/js/tests/umask_test.ts

```diff
@@ -11,6 +11,6 @@ unitTest(
     const finalMask = Deno.umask();
     assertEquals(newMask, 0o020);
     assertEquals(finalMask, prevMask);
-    assertEquals(prevMask, 0o022);
+    assert(prevMask & 0o002);
```

Comment:
Is this assert useful? Can't umasks be 0? I don't think this matters for Deno. I'd rather not have the assert than have to change this again.